cinema/views.py

```diff
@@ -1 +1,63 @@
-# write views here
+from rest_framework import viewsets
+
+from cinema.models import CinemaHall, Genre, Actor, Movie, MovieSession
+from cinema.serializers import (
+    CinemaHallSerializer,
+    GenreSerializer,
+    ActorSerializer,
+    MovieSerializer,
+    MovieSessionSerializer,
+    MovieListSerializer,
+    MovieRetrieveSerializer,
+    MovieSessionListSerializer,
+    MovieSessionRetrieveSerializer,
+)
+
+
+class CinemaHallViewSet(viewsets.ModelViewSet):
+    queryset = CinemaHall.objects.all()
```

Comment:
Avoid using `.all()` before `.prefetch_related()` in the `queryset` definition. Instead, directly use `CinemaHall.objects.prefetch_related(...)` if needed, or define the `queryset` in the `get_queryset` method.